server/apps/postgres-app/src/main/java/org/apache/james/PostgresJamesServerMain.java

```diff
@@ -114,8 +114,6 @@ public static void main(String[] args) throws Exception {
 
     static GuiceJamesServer createServer(PostgresJamesConfiguration configuration) {
         return GuiceJamesServer.forConfiguration(configuration)
-            .combineWith(POSTGRES_MODULE_AGGREGATE)
-            .combineWith(new UsersRepositoryModuleChooser(new JPAUsersRepositoryModule())
-                .chooseModules(configuration.getUsersRepositoryImplementation()));
+            .combineWith(POSTGRES_MODULE_AGGREGATE);
```

Comment:
We need to keep the user module chooser IMO.

```suggestion
            .combineWith(new UsersRepositoryModuleChooser(new PostgresUsersRepositoryModule())
                .chooseModules(configuration.getUsersRepositoryImplementation()));
```

This should solve the failing test `userFromLdapShouldLoginViaImapProtocol(GuiceJamesServer) – org.apache.james.PostgresWithLDAPJamesServerTest`